.solcover.js

```diff
@@ -1,14 +1,13 @@
-const skipFiles = [
-  'lib',
-  'test',
-  'acl/ACLSyntaxSugar.sol',
-  'common/DepositableStorage.sol', // Used in tests that send ETH
-  'common/UnstructuredStorage.sol' // Used in tests that send ETH
-]
-
 module.exports = {
-    norpc: true,
-    compileCommand: '../node_modules/.bin/truffle compile',
-    testCommand: 'node --max-old-space-size=4096 ../node_modules/.bin/truffle test --network coverage',
-    skipFiles,
+  skipFiles: [
+    'lib',
+    'test',
+    'acl/ACLSyntaxSugar.sol',
+    'common/DepositableStorage.sol', // Used in tests that send ETH
+    'common/UnstructuredStorage.sol' // Used in tests that send ETH
```

Comment:
I think we might be able to remove these from the skipped files now since gas is less of an issue; worth a try?

Reply:
Let's do it!